autoload/neomake/makers/ft/d.vim

```diff
@@ -49,8 +49,7 @@ function! neomake#makers#ft#d#dmd()
     return {
         \ 'args': l:args,
         \ 'errorformat':
-        \     '%-G%f:%s:,%f(%l): %m,' .
-        \     '%f:%l: %m',
+        \     '%f(%l\,%c): %m,'
```

Comment:
Hi, sorry for the late reply. I'm done with it, it seems to be working just fine now.